src/helpers/graph.js

```diff
@@ -1,38 +1,46 @@
+function doesDeletedNodeHasSiblings(edges, parentId) {
+  return edges.find(edge => edge.from === parentId)
+}
+
+function revokeParentStatus(nodes, parentId) { 

```

Comment:
What does it mean to revoke a parent status?

Do nodes have status?

Reply:
(Question from the following comment from point 5: https://github.com/DarthVanger/NeuralNotes/pull/70/files#r368251724)

Reply:
Yes, we need to revoke the `parent` status of the node when we delete all its children to make sure the colors of nodes are rendered properly.

Reply:
I mean we have *node group*. There is no such thing as "parent status" in the code.

`revokeParentStatus` is not revoking the parent group. It is setting the group to `children`.

So the function name is confusing.

`revokeParentStatus` in fact sets the node group to `children`.